includes/wikia/ExternalUser_Wikia.php

```diff
@@ -210,35 +210,16 @@ public function getLastAuthenticationError() {
 	}
 
 	public function authenticate( $password ) {
-		// Log to check whether this is used anymore (it shouldn't be), so we can clean
-		// this up.
-		// @todo remove after the new password hashing is implemented (PLATFORM-2526)
-		Wikia\Logger\WikiaLogger::instance()->debug(
-			'NEW_HASHING ExternalUser authenticate called',
+		// All authentication should now go through Helios
+		Wikia\Logger\WikiaLogger::instance()->error(
+			'Unsupported ExternalUser_Wikia::authenticate called',
 			[
 				'user_id' => $this->getId(),
 				'caller' => wfGetCaller(),
 				'exception' => new Exception()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "exception" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)